apiserver/gui.go

```diff
@@ -354,7 +365,7 @@ func (h *guiHandler) serveConfig(w http.ResponseWriter, req *http.Request) {
 	w.Header().Set("Content-Type", jsMimeType)
 	tmpl := filepath.Join(h.rootDir, "templates", "config.js.go")
 	if err := renderGUITemplate(w, tmpl, map[string]interface{}{
-		"base":             h.baseURLPath,
+		"base":             guiURLPathPrefix,
```

Comment:
Actually, I think we could support also the `newer Juju/older GUI` scenario here. What about this logic?
In this example, the old-style request is */gui/uuid-1234-5678/*, the new-style one is */gui/u/admin/default*.
```
if request is old-style:
  base is /gui/uuid-1234-5678/
else if gui version < 2.3.0:
  base is /gui/u/admin/default
else:
  base is /gui/
```